model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -182,13 +182,20 @@ def nodes_builder(model: GraphModule,
         else:
             graph_node_type = BaseNode
             kwargs = {}
+
+        output_channel_index = ChannelAxis.NCHW.value
```

Comment:
This look like a patch, let's talk about this change please don't merge it until we talk @lior-dikstein, @ofirgo, @eladc-git .

Reply:
I removed it from graph builder and create a dictionary.